workflow/infrastructure-monitoring/container-monitoring.mdx

```diff
@@ -1,73 +1,131 @@
+        </Accordion>
 
--   **RX (receive) packets**: The number of packets that are received by the container over the network interface.
--   **TX (transmit) packets**: The number of packets that are transmitted by the container over the network interface.
--   **RX dropped**: The number of packets that are dropped or discarded by the container due to errors or other issues during transmission.
--   **TX dropped**: The number of packets that are dropped or discarded by the container due to errors or other issues during transmission.
+        <Accordion title="Containers">
+            Containers are the fundamental units running with pods, which are scheduled onto nodes in a Kubernetes cluster.
+        </Accordion>
 
-![](/images/infra-monitoring/c4.png)
+        <Accordion title="Deployments">
+            Deployments provide declarative updates to applications. They are a high-level abstraction that simplifies the management and scaling of applications.
+        </Accordion>
 
-### Block IO service recursive
-The container block I/O usage refers to the amount of data that is read from or written to a block device, such as a disk or storage volume, by a container. this is typically measured in terms of the number of input/output (I/O) operations and the amount of data that is transferred during those operations. The container.blockio.io_service_bytes_recursive metric key provides information about the amount of data that is read from or written to the block device in bytes, broken down by individual operations.
+        <Accordion title="ReplicaSets">
+            ReplicaSets ensure a specified number of replica pods are running at any given time. ReplicaSets can be used for horizontally scaling your applications. 
+        </Accordion>
 
-![](/images/infra-monitoring/c5.png)
+        <Accordion title="Jobs">
+            Jobs are controller objects that represent finite and short-lived tasks or batch processes that run within pods.
+        </Accordion>
+
+        <Accordion title="CronJobs">
+            CronJobs are controller objects that can be scheduled to automate task execution. CronJobs are Jobs that can be run on a periodic schedule.
+        </Accordion>
+
+        <Accordion title="DaemonSet">
+            DaemonSets are controllers that ensure a specific set of pods is running across your cluster. DaemonSets are used to run exactly one instance of a pod on each node in the cluster.
+        </Accordion>
+
+        <Accordion title="StatefulSet">
+            StatefulSets are used for managing stateful applications that are comprised of unique network identifiers, persistent storage, and specific ordering requirements for their deployment and scaling. 
+        </Accordion>
+
+        </AccordionGroup>
+
+</Accordion>
+
+# Usage Graph
+
+The Usage graph displays all of your team's Infrastructure usage data. Gain immediate insight into your stack's performance.
```

Comment:
Expand this section with another sentence summarizing what one can do with the usage Graph. General rule of thumb is to summarize the sub sections in the overview, for example,

"(first sentence)...The Usage Graph by default allows summaraziation by CPU or Memory usage, and a date range picker to drill into a specific timeframe." 

The "Gain immediate insight..." type of sentences should be replaced with a more actionable _how_ sentence. Feel free to remove those across docs whenever you see them.